examples/score-based-common-source.yaml

```diff
@@ -0,0 +1,133 @@
+# score-based-common-source.yaml - a template yaml for setting up an LR system and a validation experiment.
+#
+# This file features a score-based common-source system.
+#
+# Below, you can adapt the information to your own needs.
+#
+# You will need to specify:
+# - output folder
+# - path to data & how to parse it
+# - how to assign training and testing roles to your data
+# - the design of the lr system itself
+# - the validation experiment you run for your lr system
+#
+# When you are done, you can run it all using:
+# Usage: pdm run lir --setup examples/score-based-common-source.yaml
+#
+
+# Output will be written to the folder specified below. There will be a sub folder for each experiment. The sub folders
```

Comment:
De concepten van "output" en "experiment" zijn hier nog niet uitgelegd. Ik zou hier kort uitleggen wat voor soort output je kan verwachten (bijv. system output, visualizations, metrics and intermediate results). Dan specifiek op de structuur ingaan bij de experiment-sectie.